package/MDAnalysis/analysis/rdf.py

```diff
@@ -309,9 +197,9 @@ class InterRDF(AnalysisBase):
 
       plt.plot(rdf.results.bins, rdf.results.rdf)
 
-    The `exclusion_block` keyword allows the masking of pairs from
-    within the same molecule.  For example, if there are 7 of each
-    atom in each molecule, the exclusion mask `(7, 7)` can be used.
+    The ``exclusion_block`` keyword allows the masking of pairs from
```

Comment:
We actually use the single backticks to refer to keyword arguments (which just the de, although this may be inconsistent throughout.

Reply:
😅 o we should have docs about that because I've definitely been using double backticks for keywords because otherwise the italics is hard to read (and I use the :attr: and :meth: for other things)

Reply:
The example at https://userguide.mdanalysis.org/stable/contributing_code.html#guidelines-for-writing-docstrings use the single backticks.

Reply:
Good to know. I changed the docs and it actually also looks cleaner when one is able to distinguish between arguments and inline code snippets.